connector_ecommerce/models/product.py

```diff
@@ -42,6 +42,10 @@ def _get_tax_group_id(self):
              'system like Prestashop',
     )
 
+    @api.model
+    def _price_changed_fields(self):
+        return {'list_price', 'lst_price', 'standard_price'}
```

Comment:
price types have been removed in 9.0, in 8.0, it was actually based on the price types